app/Flag.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Query\Builder;
+use Illuminate\Database\Query\JoinClause;
+use Illuminate\Support\Facades\DB;
+
+class Flag extends Model
+{
+    /**
+     * The table associated with the model.
+     *
+     * @var string
+     */
+    protected $table = 'flags';
+
+    public $stringaction = ["Unflagged","Flagged", "Edit", "Removed"];
```

Comment:
$stringAction as per conventions we use

Also since it always seems to stay the same maybe make it a constant?

Reply:
thanks